WireUI/Sources/WireSettingsUI/Account/Backup/ViewControllers/BackupActionsHostingController.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+
+public final class BackupActionsHostingController: UIHostingController<BackupActionsView> {
+    private let viewModel: BackupActionsViewModel
+
+    public init(viewModel: BackupActionsViewModel) {
+        self.viewModel = viewModel
+        super.init(rootView: BackupActionsView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    dynamic required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
```

Comment:
I wanted to suggest to simplify the code, however, then we'd run into issues because the calling code must be annotated with `@MainActor` as well, so just ignore this please.
```suggestion
import SwiftUI

@MainActor
public func BackupActionsHostingController(viewModel: BackupActionsViewModel) -> UIViewController {
    UIHostingController(rootView: BackupActionsView(viewModel: viewModel))
}
```